daprdocs/content/en/operations/hosting/kubernetes/kubernetes-deploy.md

```diff
@@ -58,6 +58,14 @@ dapr init -k
 ✅  Success! Dapr has been installed to namespace dapr-system. To verify, run "dapr status -k" in your terminal. To get started, go here: https://aka.ms/dapr-getting-started
 ```
 
+### Install Dapr (a private Dapr Helm chart)
+
```

Comment:
You can install Dapr from a private registry if:

- You want more granular control of the Dapr Helm chart,
- You have a custom Dapr deployment, or
- You only allow for pulling Helm charts from trusted registries that are managed and maintained by your organization.